graffiti/schema/validator.go

```diff
@@ -48,7 +49,7 @@ func (v *JSONValidator) Validate(kind string, obj interface{}) error {
 	if err != nil {
 		return err
 	} else if !result.Valid() {
-		return ErrInvalidSchema
+		return fmt.Errorf("%v: %v", ErrInvalidSchema, result.Errors())
```

Comment:
I would rather make change ErrInvalidSchema to become a custom error accepting a `Errors` field so that we keep the error type

```
type ErrInvalidSchema struct {
  Errors []goschema.ResultError
}

func (e ErrInvalidSchema) String() {
    return fmt.Errorf("invalid schema: %v", e.Errors)
}
```

using it like :

```
return &ErrInvalidSchema{Errors: result.Errors()}
```